### ISSUE TITLE
"Your syncs have been enabled" notification

### ISSUE THREAD

**Issue by Gittyperson** [meganz/MEGAsync#652]
Thanks again for the excellent MEGAsync.

Please consider making this particular notification optional, or removing it completely since it is superfluous (no issues to report). It is intrusive having to see it every time a user logs on. Disabling all MEGAsync notifications is not a solution, since important and error-related notifications will be missed.

I tried to only set "High priority" notifications for MEGASync in Windows 10 Settings, but:

a) I do not know which notifications might be hidden by setting "High" instead of "Normal"
b) "Your syncs have been enabled" still appears anyway

Thanks

Windows 10 x64 21H2, MEGASync 4.6.1 64-bit

**Comment by SaiBork** [comment]
I agree with this, I actually found this post because I was looking for a way to stop it popping up on logon each time.

**Comment by vtmateos** [comment]
Thanks for the feedback. We are working on a more customisable break down of notifications by type, so you can set the ones that you are interested. It will be included in next versions.
